modules/fundamental/src/sbom/model/mod.rs

```diff
@@ -118,7 +118,13 @@ pub struct SbomPackage {
     /// CPEs identifying the package
     pub cpe: Vec<String>,
     /// License info
-    pub licenses: Option<String>,
+    pub licenses: Vec<LicenseBasicInfo>,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, ToSchema, SimpleObject)]
+pub struct LicenseBasicInfo {
+    pub(crate) license_name: String,
+    pub(crate) license_type: i32,
```

Comment:
I don't like this being an integer. IIRC we already have an enum for this, we should use it.